lesson_10/homework_10.py

```diff
@@ -0,0 +1,77 @@
+# Початкові дані імперії
+imperial_citizens = ["John", "Jane", "Bob", "Alice"]
+resources = {"gold": 100, "wood": 50, "stone": 30}
+
+
+# Додати нового жителя до імперії
+def add_citizen(name):
+    if name not in imperial_citizens:
+        imperial_citizens.append(name)
+        print(f"{name} був доданий до імперії.")
+    else:
+        print(f"{name} вже існує в імперії.")
+
+
+# Видалити жителя з імперії
+def remove_citizen(name):
+    if name in imperial_citizens:
+        imperial_citizens.remove(name)
+        print(f"{name} був видалений з імперії.")
+    else:
+        print(f"{name} не знайдений в імперії.")
+
+
+# Додати ресурси до імперії
+def add_resources(resource, amount):
+    if resource in resources:
+        resources[resource] += amount
+        print(f"Додано {amount} {resource} до ресурсів імперії.")
+    else:
+        print(f"Ресурс {resource} не знайдений.")
+
+
+# Обмін ресурсів
+def market_place(resource_in, resource_out, amount_in):
+    if resource_in not in resources or resource_out not in resources:
+        print("Некоректний ресурс.")
+        return
+
+    if resource_in == "stone" and resource_out == "gold":
+        rate = 5
+    elif resource_in == "stone" and resource_out == "wood":
+        rate = 1 / 5
+    elif resource_in == "wood" and resource_out == "gold":
+        rate = 25
+    else:
+        print("Некоректний обмін.")
+        return
```

Comment:
```suggestion
        return ???
```
Що повртає фція? Вихоить, що None